google/cloud/internal/oauth2_minimal_iam_credentials_rest.cc

```diff
@@ -71,12 +71,13 @@ MinimalIamCredentialsRestStub::GenerateAccessToken(
 }
 
 std::string MinimalIamCredentialsRestStub::MakeRequestPath(
-    GenerateAccessTokenRequest const& request) {
-  // TODO(#13422): Do not use hardcoded IAM endpoint. Use Universe Domain
-  // to build endpoint name.
+    GenerateAccessTokenRequest const& request) const {
+  auto universe_domain_status = universe_domain(Options{});
```

Comment:
we don't need to put `_status` in the variable name when it is encoded in the type. Just use `universe_domain`, or `ud` to save characters.

Reply:
I will use `ud`, because `universe_domain` won't compile.